### DISCUSSION TITLE
Suddenly, it's not working.

### DISCUSSION THREAD

**Discussion by owo441** [oobabooga/text-generation-webui#4580] [Q&A]
I don't know what update I did, but runpod won't run with it. Can you tell me how to install the old version? It was working fine yesterday, then it got weird.
![Screenshot_20231114_003423_Chrome](https://github.com/oobabooga/text-generation-webui/assets/82258733/3fdf853b-56e9-403d-8bc1-8ac8b211ef38)

**Comment by simcop2387** [comment]
That looks like the old api extension that's been removed as of 11/13.  remove --extension api from how you're running it and it should start up.  the OpenAI compatible api is now the way to do all scripting against this as an inference server.
